frontend/test/resolution/testCopyElision.cpp

```diff
@@ -181,18 +181,63 @@ static void test4() {
 }
 
 static void test5() {
-  testCopyElision("test5",
+  testCopyElision("test5a",
     R""""(
         proc test(cond: bool) {
           var x:int;
+          var z = x;
           if cond {
             var y = x;
             return;
           }
           x;
         }
     )"""",
-    {"M.test@6"});
+    {"M.test@8"});
+
+  testCopyElision("test5b",
+    R""""(
+        proc test(cond: bool) {
+          var x:int;
+          var z = x;
+          if cond {
+            var y = x;
+            return;
+          } else {
+          }
+        }
+    )"""",
+    {"M.test@8"});
+
+  testCopyElision("test5c",
+    R""""(
+        proc test(cond: bool) {
+          var x:int;
+          var z = x;
+          if cond {
+            var y = x;
+            return;
+          } else {
+            var yy = x;
+            return;
+          }
+        }
+    )"""",
+    {"M.test@8", "M.test@12"});
+
+  testCopyElision("test5d",
+    R""""(
+        proc test(cond: bool) {
+          var x:int;
+          var z = x;
+          if cond {
+            var y = x;
+            return;
+          }
+          var zz = x;
+        }
+    )"""",
+    {"M.test@8", "M.test@13"});
```

Comment:
Might be nice to put a comment saying this is the ID of `var y = x;` and the ID of `var zz = x;`